vector/src/main/java/im/vector/app/features/home/room/threads/ThreadsManager.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.home.room.threads
+
+import android.app.Activity
+import android.content.Context
+import android.text.Spanned
+import androidx.core.text.HtmlCompat
+import im.vector.app.R
+import im.vector.app.features.MainActivity
+import im.vector.app.features.MainActivityArgs
+import im.vector.app.features.settings.VectorPreferences
+import org.matrix.android.sdk.internal.database.lightweight.LightweightSettingsStorage
+import javax.inject.Inject
+
+/**
+ * The class is responsible for handling thread specific tasks
+ */
+class ThreadsManager @Inject constructor(
+        private val vectorPreferences: VectorPreferences,
+        private val lightweightSettingsStorage: LightweightSettingsStorage,
+        private val context: Context
+) {
+
+    companion object {
+        const val learnMoreUrl = "https://element.io/help#threads"
```

Comment:
Probably something to move to the `vector-config` module, maybe in a new `urls.xml` resource file

Reply:
Yes, thats what we discussed with adam